webknossos/examples/apply_merger_mode.py

```diff
@@ -0,0 +1,100 @@
+from os import environ
+from typing import Tuple, cast
+
+import fastremap
+
+import webknossos as wk
+
+
+def main() -> None:
+
+    ####################################
+    # Opening a merger mode annotation #
+    ####################################
+
+    nml = wk.Annotation.download(
+        "https://webknossos.org/annotations/Explorational/6204d2cd010000db0003db91"
+    ).skeleton
+
+    ###############################################
+    # Download and open the corresponding dataset #
+    ###############################################
+
+    dataset = wk.Dataset.download(
+        "l4_sample_dev", "scalable_minds", path="testoutput/l4_sample_dev"
```

Comment:
We need to set the url here, since the context is set to localhost during the tests (to avoid leaked data):
```suggestion
        "l4_sample_dev",
        "scalable_minds",
        path="testoutput/l4_sample_dev",
        webknossos_url="https://webknossos.org",
```